webapp/src/components/utils.jsx

```diff
@@ -64,3 +64,9 @@ export const getStyleForReactSelect = (theme) => {
         }),
     };
 };
+
+// TODO: change to use the isCombinedUserActivityPost in 'mattermost-redux/utils/post_list' when we upgrade to 5.12
+export const isCombinedUserActivityPost = (id) => {
```

Comment:
I know it's not related to this PR, but this utils file is in a bad place. Can we move this to a utils folder and split these up into logically named files? I think we should avoid an explosion of stuff in a generic utils.jsx like the webapp.

Reply:
also then we can import `utils/bla.jsx` instead of `../../`

Reply:
@crspeller good points, done 👍